docker-wrappers/SPRAS/spras_profile/config.yaml

```diff
@@ -0,0 +1,11 @@
+jobs: 30
```

Comment:
What does this control? Does it need to align with the number of rules in the Snakemake DAG? Or will Snakemake decide how to allocate rules to this number of jobs?

Reply:
I'll add a few comments to the profile, but this controls the maximum number of concurrent jobs allowed into the HTCondor queue. This number isn't hit when running the example config, but might be more useful for larger workflows across multiple data sets. That being said, I think its safe to make this number pretty big.